geode-redis/README.md

```diff
@@ -207,7 +207,7 @@ start server \
 |                    	| SUNION                                             	| CLUSTER RESET                 	|
 |                    	| SUNIONSTORE                                        	| CLUSTER SAVECONFIG            	|
 |                    	| TIME                                               	| CLUSTER SET-CONFIG-EPOCH      	|
-|                    	|                                                    	| CLUSTER SETSLOT               	|
+|                    	| UNLINK [1]                                         	| CLUSTER SETSLOT               	|
```

Comment:
We should update this docs file as well `redis_api_for_geode.html.md`. It may also be good to ask John Martin to review these doc updates.

Reply:
Good idea